pkg/target/http_test.go

```diff
@@ -158,7 +158,7 @@ func TestRetrieveHeaders(t *testing.T) {
 		t.Run(tt.Name, func(t *testing.T) {
 			testTargetConfig := &HTTPTargetConfig{
 				HTTPURL:                 "http://test",
-				ByteLimit:               1048576,
+				RequestByteLimit:        1048576,
```

Comment:
nit - `MessageByteLimit`

Reply:
Max messages isn't there either - I didn't update the tests yet, but there's a ticket for it. Just did enough to move to. implementing tests for batched http requests. :)